packages/svelte/src/config.ts

```diff
@@ -22,30 +22,23 @@ export function withSentryConfig(
   const mergedOptions = {
     ...DEFAULT_SENTRY_OPTIONS,
     ...sentryOptions,
+    componentTracking: {
+      ...DEFAULT_SENTRY_OPTIONS.componentTracking,
+      ...(sentryOptions && sentryOptions.componentTracking),
```

Comment:
this was a bug previously, not sure how we missed this, given we have a lot of tests...